Update theme to ease navigation and clarity

mikestopcontinues opened the pull request:
Just thought I'd create a draft PR to keep track of changes. At this point, it's barely any better than before, but I'll keep the branch in a working state if anyone wants to peek in and give some feedback.

Currently, I could use some updates to the content itself. I added support for `image` and `blurb` frontmatter on posts. If they're filled in, the content will appear in the post-list.

I would also suggest trimming the call for volunteers so that it's more punchy (this is important, and we need your help), even if it remains on the longer side. And tweaking the about-info into a mission statement (we are X who do Y because Z).

The next three thing I'm going to do: 
* Squash the header into a proper navbar with an email link and an rss link.
* Create a mission splash with a nice background image. I figured I'd pull something free from Pixabay. Something suggestive of Earth at large.
* Move the call for volunteers to the sidebar. (It might not fit, but we'll see where it goes.)

mikestopcontinues committed:
quick pass at updating some styles, condensing home page, enhancing post list

mikestopcontinues committed:
quick pass at updating some styles, condensing home page, enhancing post list

mikestopcontinues committed:
draft post changes to demo blurb/image in post list

mikestopcontinues committed:
draft post changes to demo blurb/image in post list

covid19readings committed:
Added blurb and image for some articles.

covid19readings committed:
Added blurb and image for some articles.

covid19readings commented:
Thanks for the quick work Mike! I just pushed a commit with image and blurb for a few articles so you can test.

covid19readings committed:
More image and blurbs.

covid19readings committed:
More image and blurbs.

mikestopcontinues committed:
more round

covid19readings committed:
Merge branch 'restyle' of github.com:covid19readings/Articles-from-China into restyle

covid19readings committed:
Shortened home message.

mikestopcontinues committed:
merge changes

mikestopcontinues commented:
Thanks for being so on top of things, @covid19readings! 

I just added columns, a favicon (it's silly, but you can replace it using a favicon generator), and a squashed header. Take a look and tell me what else you'd like to change.

BTW, I used really basic email obfuscation. Link in the source.

mikestopcontinues committed:
smaller images, fresh fonts, categories

mikestopcontinues committed:
Merge commit 'b713bda017f940de67b15ae91a8eb48dee1fcd66' into restyle

mikestopcontinues committed:
use vars for css colors

mikestopcontinues commented:
I squashed the images in the blog roll, adding the categories and moving the metadata near the titles.

I renamed the blog roll "Translations by Post Date" to clarify that the newest news will appear first.

I switched to Roboto font and I pulled a palette from colormind.io to give the site a more modern look.

Let me know what else I can do!

mikestopcontinues committed:
remove sidebar, dynamically change html title

mikestopcontinues committed:
slide images down (revertable)

mikestopcontinues committed:
Merge commit '22ae20b6f4951bccbef168facf7db9422609ff8e' into restyle

covid19readings committed:
Revert "slide images down (revertable)"

This reverts commit 225b6e9f1d08ae5d73af9afb552025848f6c1037.

covid19readings committed:
Adjustments.

mikestopcontinues committed:
bigger images, responsive

covid19readings committed:
Image and blurb for more articles

mikestopcontinues committed:
more image height on mobile

mikestopcontinues committed:
Merge branch 'restyle' of github.com:covid19readings/Articles-from-China into restyle

covid19readings committed:
Updated favicon and home page title.

covid19readings committed:
Added image and blurb for all articles.

covid19readings committed:
Added logo to footer.
